borg/key.py

```diff
@@ -126,28 +126,29 @@ class AESKeyBase(KeyBase):
     def id_hash(self, data):
         """Return HMAC hash using the "id" HMAC key
         """
-        return HMAC(self.id_key, data, sha256).digest()
+        return hmac_sha256(self.id_key, data)
```

Comment:
I don't think that it makes a big difference when changing HMACs whether we'd need to edit all usages of the HMAC as before, or edit/copy hmac_sha256 and edit all occurences as well. In both cases the work surrounding the change would be much larger (upgrader etc.)